.circleci/test-deploy.yml

```
@@ -57,12 +81,32 @@ workflows:
           working-directory: examples/angular-app
           cypress-cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/angular-app/package.json" }}
           start-command: "npm run start"
+          cypress-command: "npx cypress run --browser chrome"
+          install-browsers: true
+          node-cache-version: "v2"
+      - cypress/run:
+          filters: *filters
+          name: Run in Parallel
+          working-directory: examples/angular-app
+          cypress-cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/angular-app/package.json" }}
+          cypress-command: "npx cypress run --component --parallel --record"
+          node-cache-version: "v2"
       - install-and-persist:
           filters: *filters
           name: Install & Persist
-      - run-tests-in-parallel:
+      - run-tests-in-chrome:
           filters: *filters
-          name: Run Tests in Parallel
+          name: Run Tests in Chrome
+          requires:
+            - Install & Persist
+      - run-tests-in-firefox:
+          filters: *filters
+          name: Run Tests in Firefox
+          requires:
+            - Install & Persist
+      - run-tests-in-edge:
+          filters: *filters
+          name: Run Tests in Edge
```

We can be specific, just in case.

```suggestion
          name: Run Tests in Microsoft Edge
```